src/OrbitGl/TimerTrack.cpp

```diff
@@ -252,7 +252,7 @@ void TimerTrack::DoUpdatePrimitives(Batcher* batcher, TextRenderer& text_rendere
   draw_data.min_tick = min_tick;
   draw_data.max_tick = max_tick;
 
-  draw_data.batcher = batcher;
+  draw_data.batcher = &batcher;
```

Comment:
shouldn't this be a reference also?

Reply:
Yeah, but then `DrawData` needs a proper constructor (you can't leave a reference uninitialized), and I didn't want to start going there and refactoring `GetDrawData`...

Reply:
got it!